wiki/markdown/curl-examples.md

```diff
@@ -374,7 +374,7 @@ The back-end used is running on localhost and port 8080. EI front-end forwards t
                 {
                     "conditions":[
                         {
-                            "jmespath":"identity=='pkg:maven/com.mycompany.myproduct/artifact-name@1.0.0'"
+                            "jmespath": "split(identity, '/') | [1] =='com.mycompany.myproduct'"
```

Comment:
Just for documentation purposes (and in some of the tests also, maybe?) it would be good to have some examples where we match on the specific version or package type instead of artifact name. To get a good variation of our examples :)

Reply:
I have added a case for the sake of completion... However new methods are unit tested.

Reply:
Ok, nice I just wanted a clarification in the documentation, that hightlights different use cases. But do we not need any tests in frontend, or is it enough with the unit tests in back-end?

Reply:
Well, I dont think that we test jmespath method in the frontend... To my understanding we only test the GUI elements and bridge functionalities..i.e correct responses are communicated. Or may be I am missing something?

Reply:
No upon closer inspection I agree with you, I'm satisfied with this :)